PixiEditor/Views/Dialogs/HelloTherePopup.xaml

```diff
@@ -220,6 +220,19 @@
                 </StackPanel>
 
                 <uc:AlignableWrapPanel Grid.Row="3" HorizontalContentAlignment="Center" HorizontalAlignment="Center" Margin="0,5,0,15">
+                    <Button Command="{Binding OpenHyperlinkCommand}" CommandParameter="https://github.com/PixiEditor/PixiEditor"
```

Comment:
Can you tell me how did you test that, so you didn't notice this xD?

Reply:
uhm...